dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -323,6 +323,7 @@ class ScriptsControllerTest < ActionController::TestCase
   test 'updates project_sharing' do
     sign_in @levelbuilder
     Rails.application.config.stubs(:levelbuilder_mode).returns true
+    File.stubs(:write)
```

Comment:
Actually, I think this more specific command would be better since it enforces which files are written:
```ruby
File.stubs(:write).with {|filename, _| filename == "config/scripts/#{script.name}.script" || filename.end_with?('scripts.en.yml')}
```